src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -133,7 +133,65 @@ const timezoneLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log)
       '"logging.timezone" has been deprecated and will be removed ' +
         'in 8.0. To set the timezone moving forward, please add a timezone date modifier to the log pattern ' +
         'in your logging configuration. For more details, see ' +
-        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.md'
+        'https://github.com/elastic/kibana/blob/master/src/core/server/logging/README.mdx'
+    );
+  }
+  return settings;
+};
+
+const destLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.dest')) {
+    log(
+      '"logging.dest" has been deprecated and will be removed ' +
+        'in 8.0. To set the destination moving forward, you can use the "console" appender ' +
+        'in your logging configuration or define a custom one. For more details, see ' +
+        'https://github/elastic/kibana/blob/master/src/core/server/logging/README.mdx.'
+    );
+  }
+  return settings;
+};
+
+const quietSilentVerboseLoggingDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
```

Comment:
I initially applied the config deprecations for each of these individually. It made more sense to combine them because the the KP alternative uses the level for setting the log verbosity.